### ISSUE TITLE
False-positive for private-protected members (RCS1213)

### ISSUE THREAD

**Issue by jhinder** [JosefPihrt/Roslynator#246]
Referencing a `private protected` member from a subclass will result in a false-positive for RCS1213.

```c#
public class RCS1213ReproBase
{
    private protected void InternalImpl() // RCS1213: "Remove unused method declaration"
    { }
}

public class RCS1213Repro : RCS1213ReproBase
{
    public void DoSomething()
    {
        InternalImpl();
    }
}
```

Environment: Roslynator 1.6.20, VS 15.5.2
